content/en/flux/installation/bootstrap/gitlab.md

```diff
@@ -5,81 +5,128 @@ description: "How to bootstrap Flux with GitLab and GitLab Enterprise"
 weight: 30
 ---
 
-### GitLab and GitLab Enterprise
+The [flux bootstrap gitlab](/flux/cmd/flux_bootstrap_gitlab/) command deploys the Flux controllers
+on a Kubernetes cluster and configures the controllers to sync the cluster state from a GitLab project.
+Besides installing the controllers, the bootstrap command pushes the Flux manifests to the GitLab project
+and configures Flux to update itself from Git.
+
+After running the bootstrap command, any operation on the cluster (including Flux upgrades)
+can be done via Git push, without the need to connect to the Kubernetes cluster.
+
+{{% alert color="danger" title="Required permissions" %}}
+To bootstrap Flux, the person running the command must have **cluster admin rights** for the target Kubernetes cluster.
+It is also required that the person running the command to be the **owner** of the GitLab project,
+or to have admin rights of a GitLab group.
+{{% /alert %}}
 
-The `bootstrap gitlab` command creates a GitLab repository if one doesn't exist and
-commits the Flux components manifests to specified branch. Then it
-configures the target cluster to synchronize with that repository by
-setting up an SSH deploy key or by using token-based authentication.
+## GitLab PAT
```

Comment:
I am sharing these two resources with you. I found the information there very relevant. Can we add an external reference to this? https://docs.gitlab.com/ee/user/clusters/agent/gitops/flux_tutorial.html
https://www.youtube.com/watch?v=EjPVRM-N_PQ